lbz/misc.py

```diff
@@ -24,12 +22,6 @@ def __getitem__(self, key: str) -> Any:
 
 
 class Singleton(type):
-    """
```

Comment:
Why remove it - this specific usage is not 100% obvious.

Reply:
I would say the declaration of this class says enough but ok, I can restore it 👍 

Removed only because of missing first line of the docstring which is required now.